### PR TITLE
Fix default config. and its doc.

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
Spawned by https://github.com/project-fifo/rebar3_lint/pull/39.

### CODE REVIEW

**Comment by paulo-ferraz-oliveira** [comment]
I would probably not have introduced this issue if https://github.com/inaka/elvis_core/issues/157 was implemented 🤣

File: README.md [hunk 1]
```diff
@@ -51,9 +51,6 @@ This is the default configuration if no input is provided:
 [#{ dirs => ["apps/*/src/**", "src/**"],
     filter => "*.erl",
     ruleset => erl_files },
- #{ dirs => ["."],
```
Comment (paulo-ferraz-oliveira) [inline README.md:54]: 😄

File: elvis.config [hunk 2]
```diff
@@ -0,0 +1 @@
+[].
```
Comment (paulo-ferraz-oliveira) [inline elvis.config:1]: This prevents a warning, but it's not 100% required (except for that <<<).

File: rebar.config [hunk 3]
```diff
@@ -45,16 +45,3 @@
 %% == Aliases ==
 
 {alias, [{test, [xref, dialyzer, hank, ct, cover, edoc]}]}.
-
```
Comment (paulo-ferraz-oliveira) [inline rebar.config:48]: One less source of error, now moved to the test suites.

File: src/rebar3_lint_prv.erl [hunk 4]
```diff
@@ -80,7 +81,9 @@ try_elvis_config_file(State) ->
     Filename = filename:join(rebar_dir:root_dir(State), "elvis.config"),
     rebar_api:debug("Looking for Elvis in ~s", [Filename]),
     try elvis_config:from_file(Filename) of
-        [] -> default_config();
+        [] ->
```
Comment (paulo-ferraz-oliveira) [inline src/rebar3_lint_prv.erl:84]: I was actually thinking about formatting this with `rebar3_format`. Let me know.

File: src/rebar3_lint_prv.erl [hunk 5]
```diff
@@ -90,13 +93,12 @@ try_elvis_config_file(State) ->
 
 -spec default_config() -> elvis_config:configs().
 default_config() ->
-    rebar_api:debug("Using default Elvis configuration", []),
```
Comment (paulo-ferraz-oliveira) [inline src/rebar3_lint_prv.erl:93]: Moved to a context where `rebar_api` is actually available.

File: README.md (hunk 1 above)
Comment (elbrujohalcon) [reply README.md:54]: Oh, the good old times of `erlang.mk` 👴🏻

File: src/rebar3_lint_prv.erl (hunk 4 above)
Comment (elbrujohalcon) [reply src/rebar3_lint_prv.erl:84]: Go for it! Probably on a different PR.

**Review by elbrujohalcon — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (paulo-ferraz-oliveira) [reply README.md:54]: 👍

**Pull request merged**